src/js/cf-tables.js

```diff
@@ -0,0 +1,162 @@
+
+/**
+ * cf-tables
+ * https://github.com/cfpb/cf-tables
+ *
+ * A public domain work of the Consumer Financial Protection Bureau
+ */
+
+( function( $ ) {
+  'use strict';
+
+  var SortableTable = function( table, options ) {
+    /*  At the moment, there are no default settings, but here's an object
+        for their future use! */
+    var defaults = {
+        },
```

Comment:
No need for a line break, should make this consistent with the empty object below.